test/attr-for-selected.html

```diff
@@ -158,6 +168,21 @@
         testSelectItem(i);
       });
     });
+
+    suite('default attribute', function() {
+      var selector;
+      var items;
+
+      setup(function () {
+        selector = fixture('defaultAttribute');
+        items = Array.prototype.slice.apply(selector.querySelectorAll('div[some-attr]'));
+      });
+
+      test('setting non-existing value sets default', function() {
+        selector.select('non-existing-value');
+        assert.equal(selector.selectedItem, items[2]);
```

Comment:
I will add an assert that the `selector.selected` was not modified.

About the change of the assert: I used the format of the other tests in the file to remain consistent. Therefore I think it is better to keep it like this or else update all tests in this file.